magstock/templates/regextra.html

```diff
@@ -0,0 +1,116 @@
+
+<script type="text/javascript">
+    if (window.BADGE_TYPES) {
+        BADGE_TYPES[0].description = 'Includes three day admission and campground fees';
+        BADGE_TYPES[1].description = 'T-shirt level plus additional swag. (Available until {{ c.SUPPORTER_DEADLINE|datetime:'%b %d' }}. Limited quantities.)';
+        {% if c.SHIRT_LEVEL in c.PREREG_DONATION_TIERS %}
+            BADGE_TYPES.splice(1, 0, {
+                title: 'Plus T-Shirt: ${{ c.BADGE_PRICE|add:c.SHIRT_LEVEL }}',
+                description: 'Attendee level plus a MAGStock themed t-shirt',
+                extra: {{ c.SHIRT_LEVEL }}
+            });
+        {% endif %}
+        $(function () {
+            $.field('amount_extra').parents('.form-group').hide();
```

Comment:
This is how we hide an existing row.  This is pretty standard jQuery, but theoretically we could write a function in ``common.js`` to do this, like ``removeRow('amount_extra')`` if people find that more intuitive.